src/components/BulkEnrollmentPage/index.test.jsx

```diff
@@ -1,94 +1,139 @@
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 1 }],
+});
 
 const fakeStore = {
   portalConfiguration: {
     enterpriseId: 'foo',
+    enableCodeManagementScreen: false,
+    enableSubscriptionManagementScreen: true,
   },
 };
 const mockStore = configureMockStore([thunk]);
 const BulkEnrollmentWrapper = () => (
-  <MemoryRouter>
-    <Provider store={mockStore(fakeStore)}>
-      <BulkEnrollmentPage />
-    </Provider>
-  </MemoryRouter>
+  <Provider store={mockStore(fakeStore)}>
+    <BulkEnrollmentPage />
+  </Provider>
 );
 
+LicenseManagerApiService.fetchSubscriptions.mockImplementation(() => subscriptions);
+LicenseManagerApiService.fetchSubscriptionUsersOverview.mockImplementation(() => mockSubscriptionCount);
+
 describe('<BulkEnrollmentPage />', () => {
-  it('renders the bulk enrollment component', async () => {
-    renderWithRouter(<BulkEnrollmentWrapper />);
-    screen.getByText('Subscription Enrollment');
-    await act(() => subscriptions);
-  });
-  it.skip('renders the subscription picker', async () => {
-    renderWithRouter(<BulkEnrollmentWrapper />, { route: `/sluggo/admin/${ROUTE_NAMES.bulkEnrollment}` });
-    expect(await screen.findByText('Cohorts')).toBeInTheDocument();
-    expect(await screen.findByText(sub1.title)).toBeInTheDocument();
-    expect(await screen.findByText(sub2.title)).toBeInTheDocument();
+  describe('multiple subscriptions', () => {
+    it('renders the bulk enrollment component', async () => {
+      renderWithRouter(<BulkEnrollmentWrapper />);
+      screen.getByText('Subscription Enrollment');
+      await act(() => subscriptions);
+    });
+    it('renders the subscription picker', async () => {
+      renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}` });
+      expect(await screen.findByText('Cohorts')).toBeInTheDocument();
+      expect(await screen.findByText(sub1.title)).toBeInTheDocument();
+      expect(await screen.findByText(sub2.title)).toBeInTheDocument();
+    });
+    it('renders the course search page', async () => {
+      renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}/${sub1.uuid}` });
+      expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent(ADD_COURSES_TITLE);
+    });
+    it('renders course search page when navigating via subscription picker', async () => {
+      renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}` });
+      await act(() => subscriptions);
+      // The expiration modal makes the enroll learners link inaccessible, so we need to dismiss it first.
+      const modalButton = await screen.getByRole('button', { name: 'OK' });
+      userEvent.click(modalButton);
 
-    await act(() => subscriptions);
+      const sub1Button = screen.getAllByRole('link', { name: 'Enroll learners' })[0];
```

Comment:
ah so this is because we get multiple 'Enroll learners' buttons right? yeah I had asked Jen about having a button with a single enroll action on top looks like we removed that so this is fine for now

Reply:
For each subscription we have an Enroll learners button yeah. I struggled for a while trying to basically link 2 screen finds together (ie "get the enroll button under this other div/heading/etc.") and there's a "get closest <element> to" function, but there wasn't an obvious or accessible role/name/etc. for the actual names of the subscriptions to narrow this down.

TL;DR yeah I just picked one at random because there's multiples for now.

Reply:
yeah I am not too big a fan of not having the top level bulk button.. I liked having it personally.. but this is more a Jen chat than here... so all good